platform/Cargo.toml

```diff
@@ -7,6 +7,9 @@ members = [
 
 resolver = "2"
 
+[workspace.lints.rust.unexpected_cfgs]
+level = "forbid"
```

Comment:
I doubt this would work without the `check-cfg` attribute, see [here](https://doc.rust-lang.org/rustc/lints/listing/warn-by-default.html#explanation-90)

Reply:
Tested it just now locally by adding `#[cfg(abc)]` and it reported it as expected.